admin-dev/themes/default/package.json

```diff
@@ -1,11 +1,25 @@
-{ "name": "prestashop-default-theme",
-  "author": "PrestaShop",
+{
+  "name": "default",
   "version": "1.0.0",
-  "description": "install admin theme dependencies",
-  "dependencies": {
-      "PrestaKit": "https://github.com/PrestaShop/prestashop-ui-kit"
-  },
+  "description": "",
+  "main": "index.js",
   "scripts": {
-      "install": "cp node_modules/PrestaKit/dist/js/*.js js; cp node_modules/PrestaKit/dist/css/*.css css; cp node_modules/PrestaKit/dist/fonts/* fonts"
+    "postinstall": "./node_modules/.bin/webpack --progress",
```

Comment:
You don't need the "./node_modules/.bin" part, as `npm` executes commands with the node_module's `bin` dir at the front of the executable path